frontend/apps/marketing/src/components/tabGroup/TabGroup.tsx

```diff
@@ -0,0 +1,89 @@
+'use client';
+import {BaseEntry} from 'contentful';
```

Comment:
What does this do? (And can it be helpful for ActionBlock? 😅)

Reply:
Ok cool! I saw Artem used [EntryFields](https://contentful.github.io/contentful.js/contentful/10.3.4/modules/EntryFields.html) in the IconHighlight, which seems like a field-specific typing structure?